docs/ML_DGA.md

```diff
@@ -0,0 +1,175 @@
+# Machine Learning on Domain Generation Algorithm (DGA)
+
+Several blogs were put out on how you can create and leverage supervised DGA ML models to enrich data within the stack.
+* Part 1: [Machine learning in cybersecurity: Training supervised models to detect DGA activity](https://www.elastic.co/blog/machine-learning-in-cybersecurity-training-supervised-models-to-detect-dga-activity)
+* Part 2: [Machine learning in cybersecurity: Detecting DGA activity in network data](https://www.elastic.co/blog/machine-learning-in-cybersecurity-detecting-dga-activity-in-network-data)
+
+You can also find some supplementary and examples [here](https://github.com/elastic/examples/tree/master/Machine%20Learning/DGA%20Detection)
+
+For questions, please reach out to the ML team in the #machine-learning channel of the 
+[Elastic public slack channel](https://www.elastic.co/blog/join-our-elastic-stack-workspace-on-slack)
+
+They can also be reached by using the `stack-machine-learning` tag in the [discuss forums](https://discuss.elastic.co/tags/c/elastic-stack/stack-machine-learning)
+
+*Note: in order to use these ML features, you must have a platinum or higher [subscription](https://www.elastic.co/subscriptions)*
```

Comment:
Hey @brokensound77 !  Really sorry for the late comment, but could we add a note here indicating that the . Stack ML components that this runs on are experimental?

Reply:
good call, added in fffac11d7991f486bcda52bd8ceca248394e4ae8